pkg/cloud/cloud.go

```diff
@@ -108,13 +108,13 @@ func NewCloud() (Cloud, error) {
 	api, err := DefaultKubernetesAPIClient()
 
 	if err != nil && !isDriverBootedInECS() {
-		return nil, fmt.Errorf("could not instantiate k8s api client: %v", err)
+		klog.Warning("Could not create Kubernetes Client")
```

Comment:
:+1: but the is error not being printed anymore

also, NewCloudWithRole needs the same update (better yet, refactored so NewCloudWithRole and NewCloud call the same internal function so since they only differ by one line)